data/campaigns/Winds_of_Fate/scenarios/10_Fire_Meets_Steel.cfg

```diff
@@ -748,33 +737,33 @@ We shall find no better neighbors than each other."
             [/filter]
             variable=loc
         [/store_unit]
-        # Move Reshan to that hex.
+        # Move Resha to that hex.
         [move_unit]
-            id=Reshan
+            id=Resha
             to_x,to_y=$loc.x,$loc.y
         [/move_unit]
         # Set facings assuming most likely positions of the units.
-        {MODIFY_UNIT id=Reshan facing se}
+        {MODIFY_UNIT id=Resha facing se}
         {MODIFY_UNIT id=Gorlack facing nw}
         {CLEAR_VARIABLE loc}
         [message]
-            speaker=Reshan
+            speaker=Resha
             message= _ "Gorlack, it seems no cave can keep us!"
         [/message]
         [message]
             speaker=Gorlack
             image_pos=right
             mirror=yes
-            message= _ "reshan...
+            message= _ "resha...
```

Comment:
Capitalize Resha?

Reply:
Hmm, not sure... was trying to give it the feeling of being said very quietly and softly in disbelief. Wanted to make the tonality sound unique in some way since it is meant to be a turning point for this character. What is your sense on it?

Reply:
I figured that was your idea but to me it comes across as just a typo since everything else is capitalized. This is just current grammatical conventions of course, but just "..." suffices seeing that Gorlack is pretty decisive otherwise. You could even add two of them like "... Resha......" for emphasis if you really wanted. Again mostly just following current conventions.

Reply:
I don't think making it lowercase gives a sense of being said softly. The prefix `...` might do that, or maybe making it a smaller font size?

Reply:
Alright, capitalized it. The smaller font size is a good idea but was a bit too hard to read, so instead I gave the following exclaimed line a larger font size.